src/dodal/devices/pressure_jump_cell.py

```diff
@@ -126,30 +107,30 @@ def __init__(
                 )
             )
 
-        self.fast_valve_control: DeviceVector[SignalRW[FastValveControlRequest]] = (
-            DeviceVector(
-                {
-                    i: epics_signal_rw(FastValveControlRequest, f"{prefix}V{i}:CON")
-                    for i in self.fast_valves
-                }
-            )
+        self.fast_valve_control: DeviceVector[FastValveControl] = DeviceVector(
+            {i: FastValveControl(f"{prefix}V{i}") for i in self.fast_valves}
         )
 
-        self.valve_control: DeviceVector[SignalRW[ValveControlRequest]] = DeviceVector(
-            {
-                i: epics_signal_rw(ValveControlRequest, f"{prefix}V{i}:CON")
-                for i in self.slow_valves
-            }
+        self.valve_control: DeviceVector[ValveControl] = DeviceVector(
+            {i: ValveControl(f"{prefix}V{i}") for i in self.slow_valves}
         )
 
         super().__init__(name)
 
     async def set_valve(
-        self, valve: int, value: ValveControlRequest | FastValveControlRequest
+        self,
+        valve: int,
+        value: ValveControlRequest | FastValveControlRequest | ValveOpenSeqRequest,
```

Comment:
As Peter commented in #759 that only OPENSEQ is used for opening the valves, `set_valve()` could map `OPEN` in `ValveControlRequest` and `FastValveControlRequest` to `ValveOpenSeqRequest.OPEN_SEQ` so when using `set_valve()` you don't have to know to use OPEN_SEQ.

Reply:
good UX!

Reply:
fixed in 03642dcaf55c97379809911ece94dd58089a5824